yorkie/config.go

```diff
@@ -14,172 +14,100 @@
-func NewConfigFromFile(path string) (*Config, error) {
-	conf := &Config{}
-	file, err := os.Open(filepath.Clean(path))
-	if err != nil {
-		log.Logger.Error(err)
-		return nil, err
-	}
-
-	if err := json.NewDecoder(file).Decode(conf); err != nil {
-		log.Logger.Error(err)
-		return nil, err
-	}
-
-	conf.ensureDefaultValue()
-
-	return conf, nil
-}
-
-// RPCAddr returns the RPC address.
-func (c *Config) RPCAddr() string {
-	return fmt.Sprintf("localhost:%d", c.RPC.Port)
-}
-
-// Validate returns an error if the provided Config is invalidated.
-func (c *Config) Validate() error {
-	if err := c.RPC.Validate(); err != nil {
-		return err
-	}
-
-	// TODO(umi0410): Other validations will be here later.
-
-	if err := c.Backend.Validate(); err != nil {
-		return err
-	}
-
-	if err := c.Mongo.Validate(); err != nil {
-		return err
-	}
-
-	if c.ETCD != nil {
-		return c.ETCD.Validate()
-	}
-	return nil
-}
-
-// ensureDefaultValue sets the value of the option to which the default value
-// should be applied when the user does not input it.
-func (c *Config) ensureDefaultValue() {
-	if c.RPC.Port == 0 {
-		c.RPC.Port = DefaultRPCPort
-	}
-
-	if c.Metrics.Port == 0 {
-		c.Metrics.Port = DefaultMetricsPort
-	}
-
-	if c.Mongo.ConnectionTimeout == "" {
-		c.Mongo.ConnectionTimeout = DefaultMongoConnectionTimeout.String()
-	}
-
-	if c.Mongo.ConnectionURI == "" {
-		c.Mongo.ConnectionURI = DefaultMongoConnectionURI
-	}
-
-	if c.Mongo.YorkieDatabase == "" {
-		c.Mongo.YorkieDatabase = DefaultMongoYorkieDatabase
-	}
-
-	if c.Mongo.PingTimeout == "" {
-		c.Mongo.PingTimeout = DefaultMongoPingTimeout.String()
-	}
-
-	if c.Backend.SnapshotThreshold == 0 {
-		c.Backend.SnapshotThreshold = DefaultSnapshotThreshold
-	}
-
-	if c.Backend.SnapshotInterval == 0 {
-		c.Backend.SnapshotInterval = DefaultSnapshotInterval
-	}
-
-	if c.Backend.AuthWebhookMaxRetries == 0 {
-		c.Backend.AuthWebhookMaxRetries = DefaultAuthWebhookMaxRetries
-	}
-
-	if c.Backend.AuthWebhookMaxWaitInterval == "" {
-		c.Backend.AuthWebhookMaxWaitInterval = DefaultAuthWebhookMaxWaitInterval.String()
-	}
-
-	if c.Backend.AuthWebhookCacheAuthTTL == "" {
-		c.Backend.AuthWebhookCacheAuthTTL = DefaultAuthWebhookCacheAuthTTL.String()
-	}
-
-	if c.Backend.AuthWebhookCacheUnauthTTL == "" {
-		c.Backend.AuthWebhookCacheUnauthTTL = DefaultAuthWebhookCacheUnauthTTL.String()
-	}
-}
-
-func newConfig(port int, metricsPort int, dbName string) *Config {
-	return &Config{
-		RPC: &rpc.Config{
-			Port: port,
-		},
-		Metrics: &prometheus.Config{
-			Port: metricsPort,
-		},
-		Backend: &backend.Config{
-			SnapshotThreshold: DefaultSnapshotThreshold,
-			SnapshotInterval:  DefaultSnapshotInterval,
-		},
-		Mongo: &mongo.Config{
-			ConnectionURI:     DefaultMongoConnectionURI,
-			ConnectionTimeout: DefaultMongoConnectionTimeout.String(),
-			PingTimeout:       DefaultMongoPingTimeout.String(),
-			YorkieDatabase:    dbName,
-		},
-	}
-}
+ package yorkie
```

Comment:
It looks like every line in the file has a single space added. Can you please check the editor settings?
```suggestion
package yorkie
```